product_brand/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2009 NetAndCo (<http://www.netandco.net>).
+# Copyright 2011 Akretion Benoît Guillot <benoit.guillot@akretion.com>
+# Copyright 2014 prisnet.ch Seraphine Lantible <s.lantible@gmail.com>
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2018 Daniel Campos <danielcampos@avanzosc.es>
+# Copyright 2018 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Brand Manager',
+    'version': '12.0.1.0.0',
+    'category': 'Product',
```

Comment:
@ThomasBinsfeld Can you set the `development_status`key?

Reply:
Done